tests/openvino/test_exporters_cli.py

```diff
@@ -84,14 +83,13 @@ class OVCLIExportTestCase(unittest.TestCase):
         ("latent-consistency", 50, 135),
     )
 
-    SUPPORTED_4BIT_ARCHITECTURES = (("text-generation-with-past", "opt125m"),)
-
-    SUPPORTED_4BIT_OPTIONS = ["int4_sym_g128", "int4_asym_g128", "int4_sym_g64", "int4_asym_g64"]
-
-    TEST_4BIT_CONFIGURATONS = []
-    for arch in SUPPORTED_4BIT_ARCHITECTURES:
-        for option in SUPPORTED_4BIT_OPTIONS:
-            TEST_4BIT_CONFIGURATONS.append([arch[0], arch[1], option])
+    TEST_4BIT_CONFIGURATONS = [
+        ("text-generation-with-past", "opt125m", "int4_sym_g128", 62, 86),
+        ("text-generation-with-past", "opt125m", "int4_asym_g128", 62, 86),
+        ("text-generation-with-past", "opt125m", "int4_sym_g64", 62, 86),
+        ("text-generation-with-past", "opt125m", "int4_asym_g64", 62, 86),
+        ("text-generation-with-past", "llama_awq", "int4 --ratio 1.0 --sym --group-size 16 --all-layers", 0, 32),
```

Comment:
Had to test `--all-layers` on a different model because `opt125m` fails at compilation when all layers are quantized into INT4

Reply:
You can try to test it like this to keep the tests light: https://github.com/huggingface/optimum-intel/blob/6faf44562d762ca66bb1db93c9c2703e37cf5020/tests/openvino/test_quantization.py#L191

Reply:
Thanks! I actually had the same idea, but the model here also has the same shortcoming. The reason the test does not fail there is because the compressed model is not being compiled. `llama_awq` is an alias for `HuggingFaceH4/tiny-random-LlamaForCausalLM` so this model is also small one.